packages/react-frontend/src/pages/SignUpPage/SignUpPage.jsx

```diff
@@ -12,6 +12,32 @@ function LoginPage({ showToast }) {
     const [btnEnabled, setBtnEnabled] = useState(true);
     const navigate = useNavigate();
 
+    useEffect(() => {
+        const script = document.createElement("script");
+        script.src = "https://smtpjs.com/v3/smtp.js";
+        script.async = true;
+        document.body.appendChild(script);
+
+        return () => {
+            document.body.removeChild(script);
+        };
+    }, []);
+    
+    const sendEmail = () => {
+        window.Email.send({
+            SecureToken: "476bb72c-c256-4a0d-b624-0cea4d2f9147",
```

Comment:
Should this token be public in our code?

Reply:
the token is hiding the username and password of the email address that is the sender